src/attribution/onINP.ts

```diff
@@ -117,9 +117,7 @@ const groupEntriesByRenderTime = (entry: PerformanceEventTiming) => {
 
   // Iterate over all previous render times in reverse order to find a match.
   // Go in reverse since the most likely match will be at the end.
-  for (let i = pendingEntriesGroups.length - 1; i >= 0; i--) {
-    const potentialGroup = pendingEntriesGroups[i];
-
+  for (const potentialGroup of pendingEntriesGroups) {
```

Comment:
Ahh, we probably can't make this change because the old code iterated in reverse.